packages/zip.rb

```diff
@@ -2,45 +2,35 @@
   source_sha256 'f0e8bb1f9b7eb0b01285495a2699df3a4b766784c1765a8f1aeedf63c0806369'
 
-  binary_url ({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/zip/3.0-11_armv7l/zip-3.0-11-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/zip/3.0-11_armv7l/zip-3.0-11-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/zip/3.0-11_i686/zip-3.0-11-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/zip/3.0-11_x86_64/zip-3.0-11-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '1905c224b2b73e81c3622c3f440540953bff6516d514addd1f1e09a75573d34b',
-     armv7l: '1905c224b2b73e81c3622c3f440540953bff6516d514addd1f1e09a75573d34b',
-       i686: '58535613e27ac7494cdfbae25d61c4971ee1d670816169a52eec79df3645e21d',
-     x86_64: 'c78a63d3630f3dbf637127e1a5a08942fd1da213cde296d33e77823e8547a56c',
-  })
+  depends_on 'bz2' => :build
 
-  # adapted from the homebrew recipe as seen at: https://github.com/Homebrew/homebrew-core/blob/master/Formula/zip.rb
-  # Upstream is unmaintained so we use the Debian patchset:
-  # https://packages.debian.org/sid/zip
-  def self.patch
-    patch_url = "https://mirrors.ocf.berkeley.edu/debian/pool/main/z/zip/zip_3.0-11.debian.tar.xz"
-    patch_sha256 = "c5c0714a88592f9e02146bfe4a8d26cd9bd97e8d33b1efc8b37784997caa40ed"
+  extra_url "https://httpredir.debian.org/debian/pool/main/z/zip/zip_#{@_ver}.debian.tar.xz" => '522174080773f72882bd240ca384e698134f61ad6405ce8f995e1d21c9ba41d8'
 
-    system('curl -#L', patch_url, '-o', 'zippatches.tar.xz')
-    abort 'Checksum mismatch :/ try again' unless Digest::SHA256.hexdigest( File.read("./zippatches.tar.xz") ) == patch_sha256
-    system("tar","-xf","zippatches.tar.xz")
+  def self.patch
+    system "tar xf zip_#{@_ver}.debian.tar.xz"
+    system "for patch in $(cat debian/patches/series); do patch -p 1 -i debian/patches/${patch}; done"
 
-    system("for i in `cat debian/patches/series`; do patch -p 1 < debian/patches/$i; done")
+    system "sed -i 's:CC = cc:CC = #{CREW_TGT}-gcc #{CREW_COMMON_FLAGS}:' unix/Makefile"
+    system "sed -i 's:CPP = /lib/cpp:CPP = $(which cpp):' unix/Makefile"
+    system "sed -i 's:INSTALL_PROGRAM = cp:INSTALL_PROGRAM = install:' unix/Makefile"
+    system "sed -i 's:INSTALL_D = mkdir -p:INSTALL_D = install -d:' unix/Makefile"
+    system "sed -i 's:prefix = /usr/local:prefix = #{CREW_PREFIX}:' unix/Makefile"
+    system "sed -i 's:CFLAGS = -O2:CFLAGS =:' unix/Makefile"
   end
 
   def self.build
     system 'make -f unix/Makefile generic'
   end
 
   def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", "BINDIR=#{CREW_DEST_PREFIX}/bin", "MANDIR=#{CREW_DEST_MAN_PREFIX}/man1", '-f', 'unix/Makefile', 'install'
+    system "make", "DESTDIR=#{CREW_DEST_DIR}", "BINDIR=#{CREW_DEST_PREFIX}/bin", "MANDIR=#{CREW_DEST_PREFIX}/share/man/man1", "-f", "unix/Makefile", "install"
```

Comment:
Why was `#{CREW_DEST_MAN_PREFIX}` changed to `#{CREW_DEST_PREFIX}/share/man`

Reply:
It was copied from `unzip`, I'll fix it in the next commit.